.github/workflows/push-to-prod-draft.yml

```diff
@@ -10,34 +10,6 @@ jobs:
       fail-fast: false
       matrix:
         sbc: [raspi, rockpi, rak, pisces, og, sensecap, finestra, controllino]
-        frequency: [470, 868, 915]
```

Comment:
Do we really want to touch the prod workflows? Won't those be unchanged?

Reply:
We're removing indoor outdoor variants and all frequencies. So, not needed to keep all of matrix rows.